templates/components/cart/content.html

```diff
@@ -19,13 +19,17 @@
                             title="{{lang 'cart.gift_certificate'}}"
                         >
                     {{else}}
-                        {{> components/common/responsive-img
```

Comment:
Do we have any ways to not remove usage of responsive-img component?

Reply:
To save usage of responsive-img component we need to write additional if statements inside src and arcset properties to use ../theme_settings.productthumb_size as fallback_size. Also we don't need to have so many srcset values here, because the size of cart item image depends on its wrapper size, not the device screen width. Also this responsive-img component is used in a lot of places, that is one more reason not to make its logic more difficult